### ISSUE TITLE
pl-cz--ii-153--001-005.krn rhythm problem

### ISSUE THREAD

**Issue by craigsapp** [pl-wnifc/humdrum-polish-scores#1]
Score https://verovio.humdrum.org/?file=https%3A%2F%2Fhumdrum.nifc.pl%2Fpl-cz--ii-153--001-005.krn

has a rhythm error on line 1816, m150, where one of the notes in the grace note line is a triplet 16th (that note should probably be converted to a grace note).   The problem is in the first violin part (`24aa-`):

<img width="402" alt="Screenshot 2024-01-31 at 18 43 44" src="https://github.com/pl-wnifc/humdrum-polish-scores/assets/3487289/b4e8a3f8-fc1c-44fa-9487-3da3d06f1cfc">

**Comment by craigsapp** [comment]
The problem is that the LO:SIC is in the wrong column:

<img width="735" alt="Screenshot 2024-02-02 at 21 42 38" src="https://github.com/pl-wnifc/humdrum-polish-scores/assets/3487289/b20ad07a-4037-4152-8b0e-038845751046">

Although there is a problem still displaying the regular 24th note as a grace note.  It will probably need a layout command where the note is encoded with `q` but is displayed at regular size.

**Comment by craigsapp** [comment]
Related problem with https://verovio.humdrum.org/?file=https%3A%2F%2Fhumdrum.nifc.pl%2Fpl-kk--kk-i-8--010.krn

<img width="1386" alt="Screenshot 2024-02-08 at 01 11 15" src="https://github.com/pl-wnifc/humdrum-polish-scores/assets/3487289/90443dde-8c5f-4009-92b2-c54b2cd2fc0b">


https://polishscores.org/?id=pl-kk--kk-i-8--010

The problem is that there is a note that is `4` or `8`.  When it is `8`, there should be an invisible rest, and when `4`, that rest should be replaced with `.`.  In other words, there must be a complete rhythmic syntax in both modern and original.  For some reason `SIC:o` is used, when it should probably be `SIC:s` (i.e. the original is encoded in the `SIC` parameter, but it should be storing the correction instead.  Or what I really mean is that to `o` parameter in the SIC should be renamed to `s`.

**Comment by jacekiwaszko1** [comment]
> Score https://verovio.humdrum.org/?file=https%3A%2F%2Fhumdrum.nifc.pl%2Fpl-cz--ii-153--001-005.krn
>
> has a rhythm error on line 1816, m150, where one of the notes in the grace note line is a triplet 16th (that note should probably be converted to a grace note). The problem is in the first violin part (24aa-):

It turned out that the SIC comment should be added not to the grace note but to the following one. I moved SIC and added missing information about the beam start to the diplomatic version (gitea). Should I change the encoding here too, or will it be updated automatically?

**Comment by jacekiwaszko1** [comment]
> Related problem with https://verovio.humdrum.org/?file=https%3A%2F%2Fhumdrum.nifc.pl%2Fpl-kk--kk-i-8--010.krn

Here as well it was a mistake in SIC encoding - there should be no change of the rhythm, so each SIC note should be encoded as `8`. I fixed the encoding in the diplomatic score.

> For some reason SIC:o is used, when it should probably be SIC:s (i.e. the original is encoded in the SIC parameter, but it should be storing the correction instead. Or what I really mean is that to o parameter in the SIC should be renamed to s.

I think we decided to run all the files through the `sic -s` filter before the publication of the data. That's why `LO:SIC:o=` is used there.

**Comment by craigsapp** [comment]
> Should I change the encoding here too, or will it be updated automatically

No need to make changes here as they will be updated the next time I publish all of the score (I check new scores for basic problems and then copy all of the files to this repository).

**Comment by craigsapp** [comment]
> I think we decided to run all the files through the sic -s filter before the publication of the data. That's why LO:SIC:o= is used there.

Yes, I am vaguely remembering 😉 I think the copying process currently does not convert them, so I will add it.
